docs/compilers/Building for Core CLR.md

```diff
@@ -0,0 +1,19 @@
+Building Core-CLR-Compatible Compiler Executables
+=================================================
```

Comment:
The document should have instructions on how to build the projects.  It's lacking the correct restore step.